### ISSUE TITLE
Feature Request:   Profile features the employers might want to see. 

### ISSUE THREAD

**Issue by fluffy-hamster** [Codewars/codewars.com#835]
Okay, so here's the rub.  I'm a newbie when it comes to programming that is trying to break into the software dev industry.  I use codewars mainly for fun but I think it would be really cool if I could somehow turn my codewars activity into a way of landing a job. 

My feature request is at heart a rather simple one; to have a link to our profiles that potential employers might want to look at. For example, if potential employers could visit my profile and view my code for completed kata it might just help me (and others just like me) land their first job in the industry. Anything that can help me stand out in a positive way is worth shoving onto a CV :)

In short, my feature request is for a sharable link that I could showcase to potential employers. I can of course link my current profile, but I don't think that is gonna help that much unless they can see my code.

**Comment by jhoffner** [comment]
CW red users get a more expanded profile that is designed to showcase your skills more. There is no (and likely will never be any) way of viewing your solutions, because then the answers are exposed and the whole thing becomes meaningless (you could have just copied the solutions from somewhere else).
